Automattic/newspack-blocks
# Configure and introduce TypeScript

### All Submissions:






### Changes proposed in this Pull Request:



This PR configures and introduces [TypeScript](https://www.typescriptlang.org). As the first file to serve as an example, `src/blocks/homepage-articles/utils.ts` is used. 
What was done here was renaming it (`*.js` -> `*.ts`) and adding type information. 

### How to test the changes in this Pull Request:

1. There should be no functional changes, this is a refactoring
2. Have a look at new TS code – does it make sense?
3. Observe that the new CI step – `typescript-check` is passing

### Other information:
